docs-web/integration-with-cis.md

```diff
@@ -0,0 +1,114 @@
+
+The integration with [F5 CIS](https://clouddocs.f5.com/containers/v2/) configures an F5 BIG-IP device as a load balancer for NGINX Ingress Controller pods.
+
+> **Feature Status**: The integration with F5 CIS is available as a preview feature: it is suitable for experimenting and testing; however, it must be used with caution in production environments. Additionally, while the feature is in preview, we might introduce some backward-incompatible changes in the next releases.
+
+## Prerequisites 
+
+To enable the integration, the F5 CIS must be deployed in the cluster and configured to support the integration. Follow the instructions on the [CIS documentation portal](#link-to-be-added-later).
+
+## Configuration 
+
+### 1. Install the Ingress Controller with the Integration Enabled
+
+This step depends on how you install the Ingress Controller: using [Manifests](/nginx-ingress-controller/installation/installation-with-manifests) or the [Helm chart](/nginx-ingress-controller/installation/installation-with-helm).
+
+#### Manifests Installation
+
+1. Create a service for the Ingress Controller pods for ports 80 and 443. For example:
+    ```yaml
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: nginx-ingress-connector 
+      namespace: nginx-ingress
+      labels:
+        app: nginx-ingress-cis
+    spec:
+      ports:
+      - port: 80
+        targetPort: 80
+        protocol: TCP
+        name: http
+      - port: 443
+        targetPort: 443
+        protocol: TCP
+        name: https
+      selector:
+        app: nginx-ingress
+    ```
+    Note the label `app: nginx-ingress-cis`. We will use it in the Step 2. 
+1. In the [ConfigMap](/nginx-ingress-controller/configuration/global-configuration/configmap-resource), enable the PROXY protocol, which the BIG-IP will use to pass the client IP and port information to NGINX. For the  `set-real-ip-from` key, use the subnet of the IP, which the BIG-IP uses to send traffic to NGINX:
+    ```yaml
+    proxy-protocol: "True"
+    real-ip-header: "proxy_protocol"
+    set-real-ip-from: "0.0.0.0/0"
+    ```
+1. Deploy the Ingress Controller with additional [command-line arguments](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments):
+    ```yaml
+    args:
+    - -nginx-cis-connector=nginx-ingress
+    - -report-ingress-status 
+    . . .
+    ```
+    where `nginx-cis-connector` references the name of the NginxCisConnector resource from Step 2, and `report-ingress-status` enables [reporting Ingress statuses](/nginx-ingress-controller/configuration/global-configuration/reporting-resources-status#ingress-resources).
+
+#### Helm Installation
+
+Install a helm release with the following values that replicate the Manifest installation above:
+```yaml
+controller:
+  config:
+    entries:
+      proxy-protocol: "True"
+      real-ip-header: "proxy_protocol"
+      set-real-ip-from: "0.0.0.0/0"
+  reportIngressStatus:
+    nginxCisConnector: nginx-ingress
```

Comment:
Should this be:
`nginxCisConnector: nginx-ingress-cis` to match `app: nginx-ingress-cis` from above?